fc/parsing/actions.py

```diff
@@ -483,46 +488,55 @@ def _expr(self):
 
 
 class SetTimeUnits(BaseAction):
-    # Leaving old XML method in to document existing properties.
-    # def _xml(self):
-    #     return P.setIndependentVariableUnits(**self.get_attribute_dict('units'))
-    pass
+    """Parse action for specifying the units time should have in the model.
+
+    ``independent var units <uname>``
+    """
+    def _expr(self):
+        self.time_units = self.get_named_token_as_string('units')
+        assert self.time_units is not None
```

Comment:
Is this a sanity check, or can this show up in the user's error output?

Reply:
Should be just a sanity check - the parser would complain before now if it didn't exist.